client/jetpack-connect/plans.jsx

```diff
@@ -85,13 +87,23 @@ class Plans extends Component {
 		}
 		if ( ! this.props.canPurchasePlans ) {
 			if ( this.props.calypsoStartedConnection ) {
-				this.redirect( CALYPSO_REDIRECTION_PAGE );
+				this.redirectToCalypso();
 			} else {
 				this.redirectToWpAdmin();
 			}
 		}
 	}
 
+	redirectToCalypso() {
+		const { canPurchasePlans, selectedSiteSlug } = this.props;
+
+		if ( selectedSiteSlug && canPurchasePlans ) {
+			return this.redirect( CALYPSO_MY_PLAN_PAGE, { tour: 'main' } );
```

Comment:
Maybe add a comment that this triggers a Guided Tour -- possibly linking to the GT [README](https://github.com/Automattic/wp-calypso/blob/master/client/layout/guided-tours/README.md) since not everyone reading this code will know that the `?tour=main` qarg does.

Reply:
👍  - https://github.com/Automattic/wp-calypso/pull/22341/commits/5d3667152c26cdf6f952844b5313d0ae9a480abb

Reply:
Awesome, thanks!